crates/cheatnet/src/state.rs

```diff
@@ -446,6 +465,25 @@ impl TraceData {
     }
 }
 
+fn get_storage_values_updated(after: &CommitmentStateDiff, before: &CommitmentStateDiff) -> isize {
```

Comment:
This will be misleading and could be inaccurate when profiling for few reasons:
1. The diff is calculated in scope of the whole transaction, so that you don't pay for series of ops like `a: 1 -> a: 2`  and then later `a: 2 -> a: 1` (`a` being the storage address). This profiling metric makes sense only for a top-level call in the test IMO.
2. This doesn't seem to take into account the forking scenario, where we calculate the diff based on the chain state (above scenario applies)